### ISSUE TITLE
HomeView cannot be opened for Existing user

### ISSUE THREAD

**Issue by FilledStacks** [deepak786/intercom_flutter#142]
In my intercom service I do the following when registering the users email.

```dart
await Intercom.registerIdentifiedUser(email: email);
```

This doesn't fail through an exception but the sheet breaks. It says "something went wrong, try again" in some cases it loads forever. I've seen different messages. 

```
Tried to open HomeScreen but there was a problem initializing the SDK
```

```
Tried to open HomeScreen but a connection error occurred - 409
```

When then found out that this happens if the email being used already has an account. We us intercom on the website and we use it in the mobile apps. 

What is the process required to open up the chat on the mobile device properly with an email that already exists on the intercom backend?

**Comment by deepak786** [comment]
@FilledStacks do you use userHash to register the user in Intercom on your website?

**Comment by FilledStacks** [comment]
@deepak786 no, we go through segment to open it. I think it uses the user account details only.

**Comment by FilledStacks** [comment]
@deepak786 would we have to do that?

**Comment by deepak786** [comment]
@FilledStacks how do you register the user in your website? Please share the code.

Make sure you are using the same app id for the website and flutter app.
